src/bindings/config.ts

```diff
@@ -98,6 +99,10 @@ export const loadConfig = async (context: Context): Promise<BotConfig> => {
     wallet: {
       registerWalletWithVerification: registerWalletWithVerification,
     },
+    ask: {
+      apiKey: process.env.OPENAI_API_KEY ?? "",
```

Comment:
Pls read the openai API key from the bot's [config](https://github.com/ubiquity/ubiquibot/blob/development/src/utils/helpers.ts)